telescope/external.py

```diff
@@ -54,96 +55,100 @@ class BigQueryCommunicationError(Exception):
+                         'resources/client_secrets.json'),
+            scope='https://www.googleapis.com/auth/bigquery')
+        storage = Storage(self.credentials_filepath)
+        credentials = storage.get()
 
-    http = httplib2.Http()
-    if credentials is None or credentials.invalid:
-      credentials = run_flow(flow= flow, storage=storage, flags= GoogleAPIAuthConfig, http=http)
-      self.logger.info("Successfully authenticated with Google, moving on to building query.")
+        http = httplib2.Http()
+        if credentials is None or credentials.invalid:
+            credentials = run_flow(flow=flow,
+                                   storage=storage,
+                                   flags=GoogleAPIAuthConfig,
+                                   http=http)
+            self.logger.info(
+                "Successfully authenticated with Google, moving on to building query.")
 
-    http = credentials.authorize(http)
+        http = credentials.authorize(http)
 
-    return build('bigquery', 'v2', http=http)
+        return build('bigquery', 'v2', http=http)
 
-  def _find_project_id_opportunistically(self):
+    def _find_project_id_opportunistically(self):
 
-    authenticated_service = self.authenticate_with_google()
-    projects_handler = authenticated_service.projects()
-    projects_list = projects_handler.list().execute()
+        authenticated_service = self.authenticate_with_google()
+        projects_handler = authenticated_service.projects()
+        projects_list = projects_handler.list().execute()
 
-    if projects_list['totalItems'] == 0:
-      raise APIConfigError()
-    else:
-      project_numeric_id = projects_list['projects'][0]['numericId']
+        if projects_list['totalItems'] == 0:
+            raise APIConfigError()
+        else:
+            project_numeric_id = projects_list['projects'][0]['numericId']
 
-    return project_numeric_id
+        return project_numeric_id
 
 
 class BigQueryJobResultCollector(object):
-  """Collect the results from a BigQuery job."""
+    """Collect the results from a BigQuery job."""
 
-  def __init__(self, jobs_service, project_id):
-    """Class to collect the results from a BigQuery job when the job completes.
+    def __init__(self, jobs_service, project_id):
+        """Class to collect the results from a BigQuery job when the job completes.
 
     Args:
       jobs_service: BigQuery jobs service instance.
       project_id: (int) ID of project for which to retrieve results.
     """
-    self.logger = logging.getLogger('telescope')
-    self._jobs_service = jobs_service
-    self._project_id = project_id
+        self.logger = logging.getLogger('telescope')
+        self._jobs_service = jobs_service
+        self._project_id = project_id
 
-  def collect_results(self, job_id):
-    """Wait until a job is complete and gather all results.
+    def collect_results(self, job_id):
+        """Wait until a job is complete and gather all results.
 
     Args:
```

Comment:
Ah, good catch. It's getting this wrong. It looks like a bug in YAPF. I'll investigate.